website/docs/installation/customize.mdx

```diff
@@ -49,13 +50,22 @@ For example, to use the [jandedobbeleer][jandedobbeleer] theme, alter the init l
 oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH/jandedobbeleer.omp.json" | Invoke-Expression
 ```
 
+</TabItem>
+<TabItem value="linux">
+
+To use the [jandedobbeleer][jandedobbeleer] theme, alter the init line like this (`bash`):
+
+```bash
+eval "$(oh-my-posh init bash --config POSH_THEMES_PATH/jandedobbeleer.omp.json)"
```

Comment:
@Melyyn the linux installation doesn't create the `POSH_THEMES_PATH` environment variable so I think this will confuse people even more.